eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -78,45 +78,45 @@ jobs:
       - template: ../steps/common.yml
 
       - ${{ if ne(parameters.ServiceDirectory, '') }}:
-        - template: /eng/common/TestResources/build-test-resource-config.yml
```

Comment:
Can you undo the formatting changes as it makes it difficult to review the real changes.